src/instructlab/sdg/default_flows.py

```diff
@@ -21,6 +35,34 @@ def get_flow(self) -> list:
         pass
 
 
+class SimpleKnowledgeFlow(Flow):
+    def get_flow(self) -> list:
+        sdg_base = resources.files(__package__)
+        return [
+            {
+                "block_type": LLMBlock,
```

Comment:
@russellb This can be wrapped in an IterBlock like this https://github.com/instructlab/sdg/blob/main/src/instructlab/sdg/default_flows.py#L277-L301 and the number of samples to generate parameter can be passed on from the CLI which will inform the `num_iter` parameter so that we can run the generate block a desired number of times and generate desired no. of synthetic samples.

Reply:
thanks!